src/init.c

```diff
@@ -405,10 +409,13 @@ static void *init_stdio_handle(const char *stdio, uv_os_fd_t fd, int readable)
         break;
     case UV_NAMED_PIPE:
         handle = malloc(sizeof(uv_pipe_t));
+        JL_UV_LOCK();
         if ((err = uv_pipe_init(jl_io_loop, (uv_pipe_t*)handle, 0))) {
+            JL_UV_UNLOCK();
```

Comment:
These are not needed either.

Reply:
is it because of the unwinding as in the conversation below? Probably yes.. but isn't it less questionable this way?